### ISSUE TITLE
Env variable 

### ISSUE THREAD

**Issue by uc-asa** [best-lucky1030/Solana-Marketplace-HerosNFT#5]
I fork the code and try to follow the steps.
You  mentioned generating env value using metaplex upload but I'm getting metplex command not found the issue. Could you please help me out what are they steps needs to followed

**Comment by best-lucky1030** [comment]
@uc-asa 
which command did you fail?
could you show or share the screenshot?

**Comment by uc-asa** [comment]
Here is the comand


<img width="1792" alt="Screenshot 2022-06-09 at 12 13 46 PM" src="https://user-images.githubusercontent.com/16412197/172782035-b335e634-102b-4f8b-b89c-50c11afc5ded.png">

**Comment by best-lucky1030** [comment]
@uc-asa 
did you follow and read this doc?
https://docs.solana.com/cli/install-solana-cli-tools

**Comment by uc-asa** [comment]
Yeah but I don’t see they are using not metaplex command anywhere.

**Comment by best-lucky1030** [comment]
where did you find out that command? `metaplex upload`?

**Comment by uc-asa** [comment]
It's in your project docs that's why I asked.
<img width="1556" alt="Screenshot 2022-06-11 at 2 34 40 PM" src="https://user-images.githubusercontent.com/16412197/173181395-b5fcd635-6c5a-4319-823e-80d9ac7b86e1.png">
